### PR TITLE
9616: Standardize how multi-docketed requests are formatted

### PR DESCRIPTION
**Pull request by rachelschneiderman**
Subject case is no longer included in the list of docketNumbers provided to multi-docketing endpoints.

### CODE REVIEW

File: shared/src/business/useCases/docketEntry/addPaperFilingInteractor.ts [hunk 1]
```diff
@@ -66,6 +66,11 @@ export const addPaperFilingInteractor = async (
 
   if (!isCaseConsolidationFeatureOn) {
     consolidatedGroupDocketNumbers = [subjectCaseDocketNumber];
+  } else {
```
Comment (mrinsin) [inline shared/src/business/useCases/docketEntry/addPaperFilingInteractor.ts:69]: Nice!

**Review by mrinsin — APPROVED** [review]
Loving this refactoring!! Would it be valuable to regression test a couple of the consolidation flows with these changes?

**Comment by rachelschneiderman** [comment]
> Loving this refactoring!! Would it be valuable to regression test a couple of the consolidation flows with these changes?

Absolutely, good point. Was planning on doing that once we have edit back to a functional state, is that OK or would you prefer to do that now?

**Comment by mrinsin** [comment]
> > Loving this refactoring!! Would it be valuable to regression test a couple of the consolidation flows with these changes?
> 
> Absolutely, good point. Was planning on doing that once we have edit back to a functional state, is that OK or would you prefer to do that now?

After sounds great! Sorry I didn't realize the edit flow was not complete!TY!

**Pull request merged**